index.js

```diff
@@ -46,11 +46,10 @@ const metroQuery = async path => (await fetch(`${METRO_RETRO_URL}/${path}`, {
 
 login().then(async () => {
     const guessTemplate = (metroContent) => {
-        const metroContentSectionNames = keys(metroContent);
+        const metroContentSectionNames = keys(metroContent).filter((key) => Boolean(key));
```

Comment:
This allows the template to have zones without names (therefore without tickets inside), for decoration purpose.